Src/Scripts/generate_exceptions.py

```diff
@@ -107,6 +109,17 @@ def MakeNewException(self):
                     ExceptionInfo('FileNotFoundError', 'System.IO.FileNotFoundException', None, (), ()),
                     ExceptionInfo('PermissionError', 'System.UnauthorizedAccessException', None, (), ()),
                     ExceptionInfo('NotADirectoryError', 'IronPython.Runtime.Exceptions.NotADirectoryException', None, (), ()),
+                    ExceptionInfo('InterruptedError', 'IronPython.Runtime.Exceptions.InterruptedException', None, (), ()),
+                    ExceptionInfo('ChildProcessError', 'IronPython.Runtime.Exceptions.ChildProcessException', None, (), ()),
+                    ExceptionInfo('IsADirectoryError', 'IronPython.Runtime.Exceptions.IsADirectoryException', None, (), ()),
+                    ExceptionInfo('ProcessLookupError', 'IronPython.Runtime.Exceptions.ProcessLookupException', None, (), ()),
+                    ExceptionInfo('TimeoutError', 'IronPython.Runtime.Exceptions.TimeoutException', None, (), ()),
```

Comment:
Maybe change `IronPython.Runtime.Exceptions.TimeoutException` to `System.TimeoutException` to resolve the build issue?